position_controllers/src/joint_group_position_controller.cpp

```diff
@@ -38,14 +38,6 @@ JointGroupPositionController::init(const std::string & controller_name)
     return ret;
   }
 
-  try {
-    // undeclare interface parameter used in the general forward_command_controller
-    get_node()->undeclare_parameter("interface_name");
```

Comment:
Add something like:

```
get_node()->set_parameter("interface_name", hardware_interface::HW_IF_POSITION);
```

Reply:
setting is a very good alternative solution to this, I was a little uneasy about undeclaring anyways...

Reply:
I agree that setting it is a good idea. I've updated the branch to implement this change.